### PR TITLE
Fixed typos and small things

### PR DESCRIPTION
**Pull request by BinxiePeterson**

### CODE REVIEW

**Comment by hoytpr** [comment]
That's a great set of contributions @BinxiePeterson and we appreciate the time and careful curation of the lesson. Everything looks good to me. What do you think @raynamharris ?

File: index.md [hunk 1]
```diff
@@ -7,13 +7,13 @@ Good data organization is the foundation of any research project. It not only se
...
 
-In this lesson we'll be using data from a study of experimental evolution using *E. coli*. [More about this dataset](http://www.datacarpentry.org/organization-genomics/data/). In this study there are several types of files
+In this lesson, we'll be using data from a study of experimental evolution using *E. coli*. More information about this dataset is available [here](http://www.datacarpentry.org/organization-genomics/data/). In this study there are several types of files:
```
Comment (raynamharris) [inline index.md:10]: My only suggested change is to hyperlink more than only "here" for accessibility purposes. Something like "[More information about this dataset is available here.](http://www.datacarpentry.org/organization-genomics/data/)". I realize that linking the whole sentence seems like a lot, but it does provide more context.

**Review by raynamharris — APPROVED** [review]

**Comment by raynamharris** [comment]
This looks great! Thanks @BinxiePeterson. I made one suggestion above.

File: index.md (hunk 1 above)
Comment (hoytpr) [reply index.md:10]: Agreed.

**Comment by BinxiePeterson** [comment]
Sure thing @raynamharris I am happy with your suggestion. Thanks!

**Pull request merged**